development/pdbparse-to-json.py

```diff
@@ -21,9 +21,9 @@
     logger.addHandler(console)
 
 
-class PDBRetreiver:
+class PDBRetriever:
```

Comment:
We don't know what else taps into these classes, so you'd need to add something like:

```
class PDBRetreiver(PDBRetriever):
```
And the same for methods.  We got some wrappers in `volatility3.framework.deprecation` that should allow you to deprecate a method (and one for renaming a class).